### ISSUE TITLE
Build not working

### ISSUE THREAD

**Issue by OmgImAlexis** [bootstrap-vue/bootstrap-vue#48]
I'm trying to build a new version of this since it's not updated on npm yet. I've deleted the whole node_modules dir inside of my project and copied all the files from this repo to it. Then run `yarn install` and `yarn build`.

```
➜  bootstrap-vue git:(master) ✗ yarn install
yarn install v0.17.10
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning Unmet peer dependency "node-sass@^3.4.2".
[4/4] 📃  Building fresh packages...
success Saved lockfile.
✨  Done in 23.51s.
➜  bootstrap-vue git:(master) ✗ yarn build         
yarn build v0.17.10
$ cd scripts && ./build 
Building commonjs version
Hash: 5a7ccee4b2868cbe62ca                                                             
Version: webpack 2.1.0-beta.25
Time: 5976ms
                     Asset    Size  Chunks             Chunk Names
    bootstrapVue.common.js  192 kB       0  [emitted]  bootstrapVue
bootstrapVue.common.js.map  239 kB       0  [emitted]  bootstrapVue
    + 113 hidden modules

ERROR in bootstrapVue.common.js from UglifyJs
SyntaxError: Unexpected token: name (text) [./utils/helpers.js:5,0][bootstrapVue.common.js:79,6]
Building browser version
Hash: 3f98b00ea9d8a77eca61                                                             
Version: webpack 2.1.0-beta.25
Time: 6142ms
              Asset    Size  Chunks             Chunk Names
    bootstrapVue.js  192 kB       0  [emitted]  bootstrapVue
bootstrapVue.js.map  239 kB       0  [emitted]  bootstrapVue
    + 113 hidden modules

ERROR in bootstrapVue.js from UglifyJs
SyntaxError: Unexpected token: name (text) [./utils/helpers.js:5,0][bootstrapVue.js:78,6]
error Command failed with exit code 2.
```

**Comment by OmgImAlexis** [comment]
```
➜  bootstrap-vue git:(master) ✗ npm run build

> bootstrap-vue@0.3.12 build /Users/xo/code/Medusa-frontend/node_modules/bootstrap-vue
> cd scripts && ./build

Building commonjs version
keywords if/then/else require v5 option
Hash: 98d77844977d00159105                                                             
Version: webpack 2.1.0-beta.28
Time: 5321ms
                     Asset    Size  Chunks             Chunk Names
    bootstrapVue.common.js  193 kB       0  [emitted]  bootstrapVue
bootstrapVue.common.js.map  239 kB       0  [emitted]  bootstrapVue
    + 113 hidden modules

ERROR in bootstrapVue.common.js from UglifyJs
SyntaxError: Unexpected token: name (text) [./utils/helpers.js:5,0][bootstrapVue.common.js:82,6]
Building browser version
keywords if/then/else require v5 option
Hash: 9eb4fd9f4a6ed3cc6326                                                             
Version: webpack 2.1.0-beta.28
Time: 5657ms
              Asset    Size  Chunks             Chunk Names
    bootstrapVue.js  193 kB       0  [emitted]  bootstrapVue
bootstrapVue.js.map  239 kB       0  [emitted]  bootstrapVue
    + 113 hidden modules

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (250 kB). This can impact web performance.
Entrypoints:
  bootstrapVue (432 kB)
      bootstrapVue.js
      bootstrapVue.js.map


WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using System.import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/

ERROR in bootstrapVue.js from UglifyJs
SyntaxError: Unexpected token: name (text) [./utils/helpers.js:5,0][bootstrapVue.js:81,6]

npm ERR! Darwin 16.1.0
npm ERR! argv "/usr/local/Cellar/node/7.2.1/bin/node" "/usr/local/bin/npm" "run" "build"
npm ERR! node v7.2.1
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! bootstrap-vue@0.3.12 build: `cd scripts && ./build`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the bootstrap-vue@0.3.12 build script 'cd scripts && ./build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the bootstrap-vue package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cd scripts && ./build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bootstrap-vue
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls bootstrap-vue
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/xo/code/Medusa-frontend/node_modules/bootstrap-vue/npm-debug.log
```


Here's the `npm-debug.log`

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/Cellar/node/7.2.1/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'build' ]
2 info using npm@3.10.9
3 info using node@v7.2.1
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle bootstrap-vue@0.3.12~prebuild: bootstrap-vue@0.3.12
6 silly lifecycle bootstrap-vue@0.3.12~prebuild: no script for prebuild, continuing
7 info lifecycle bootstrap-vue@0.3.12~build: bootstrap-vue@0.3.12
8 verbose lifecycle bootstrap-vue@0.3.12~build: unsafe-perm in lifecycle true
9 verbose lifecycle bootstrap-vue@0.3.12~build: PATH: /usr/local/lib/node_modules/npm/bin/node-gyp-bin:/Users/xo/code/Medusa-frontend/node_modules/bootstrap-vue/node_modules/.bin:/Users/xo/code/Medusa-frontend/node_modules/.bin:/usr/local/Cellar/node/7.2.1/bin:/usr/local/include:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin
10 verbose lifecycle bootstrap-vue@0.3.12~build: CWD: /Users/xo/code/Medusa-frontend/node_modules/bootstrap-vue
11 silly lifecycle bootstrap-vue@0.3.12~build: Args: [ '-c', 'cd scripts && ./build' ]
12 silly lifecycle bootstrap-vue@0.3.12~build: Returned: code: 2  signal: null
13 info lifecycle bootstrap-vue@0.3.12~build: Failed to exec build script
14 verbose stack Error: bootstrap-vue@0.3.12 build: `cd scripts && ./build`
14 verbose stack Exit status 2
14 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:255:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:885:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
15 verbose pkgid bootstrap-vue@0.3.12
16 verbose cwd /Users/xo/code/Medusa-frontend/node_modules/bootstrap-vue
17 error Darwin 16.1.0
18 error argv "/usr/local/Cellar/node/7.2.1/bin/node" "/usr/local/bin/npm" "run" "build"
19 error node v7.2.1
20 error npm  v3.10.9
21 error code ELIFECYCLE
22 error bootstrap-vue@0.3.12 build: `cd scripts && ./build`
22 error Exit status 2
23 error Failed at the bootstrap-vue@0.3.12 build script 'cd scripts && ./build'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the bootstrap-vue package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     cd scripts && ./build
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs bootstrap-vue
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls bootstrap-vue
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by OmgImAlexis** [comment]
This may possibly be the issue.
https://github.com/mishoo/UglifyJS2/issues/448

**Comment by andrzejkupczyk** [comment]
@OmgImAlexis could you please provide information on which `let`s should be changed?

**Comment by pi0** [comment]
fix Will be merged & version in #50 (0.5.0) release.

**Comment by pi0** [comment]
Issue should be fixed now :)
